README.md

```diff
@@ -185,6 +196,41 @@ timestamp that denotes when the usage record was reported and
 **dimensions:** A dictionary containing the dimension names and units
 billed in the last metering.
 
+### Archive
+
+This is the metering and usage records archive. It contains a list of
+metering records with the relevent usage data, billing status, and
+timestamps. The format is as follows:
+
+```
+[
+  {
+        'billing_time': 'string(date timestamp)',
+        'billing_status': {
+            '{usage_key}': {
+                'record_id': string,
+                'status': 'succeeded'
+            }
+        },
+        'billed_usage': {
+            '{usage_key}': int
+        },
+        'usage_records': [
+            {
+                '{usage_metric}': int,
+                'reporting_time': 'string(date timestamp)',
+                'base_product': 'cpe:/o:suse:product:v1.2.3'
+            }
```

Comment:
This example kind of suggests that there is only one entry in the usage_records list... Perhaps something like would be a better representation?

```suggestion
            },
            ...
```